assignment-2/cmmi.md

```diff
@@ -1,2 +1,136 @@
+| RSKM              | Fully | RUP itself is a risk driven development process. It aims to indentify as many risks as possible and to tackle them in the early phases (elaboration) of the project.            |
+| VAL               | Fully | In RUP validation is covered by a product acceptance plan in the Project Management workflow and through beta testing during Deployment.             |
+| VER               | Fully | Verification is als covered in RUP by means of planning and conducting system and integration tests. Through its *work guideline* Review it also prescribes the practice of peer reviews.        |
+| **Level 4**|
+| OPP               | Not at all | This is not in the scope of RUP.           |
+| QPM               | Not at all | This is not in the scope of RUP.          |
+| **Level 5**        |
+| CAR               | Partially | This can be done in an Iteration Assessment. Howerver, RUP does not prescribe any analysis techniques for it.           |
+| OPM               | Not at all | This is not in the scope of RUP.             |
+
+## HealthCare.gov
+
+| Cmmi Process Area | Covered | Rationale  |
+| ------------------| ------- | ---------  |
+| **Level 2**           |
+| CM                | Unknown | There is no information found on how configuration management was processed in the HealthCare.gov project.           |
+| MA                | Not at all | Obama claimed to not be well notified about the status of the project and an audit by the Inspector General office showed that CMs did not keep adequate track of records [IHBEAT]. This indicates that there were no processes defined to measure aspects and of the project and correctly report them to the relevant parties.             |
+| PMC               | Not at all | There were a number of the issues found early in the project's lifecycle during audits such as how there was no enough time allotted to test the system. The fact that these issues even until after the launch of the site indicated that there was little to no control methods in place [POLI].             |
+| PP                | unknown | TODO: Needs more research. Probably present          |
+| PPQA              | unknown | There a number of external audits on the project but there is no information known on how CMS dealt with it internally.            |
+| REQM              | Not at all | There was a complete lack of adequate requirements tracking and management as contractors reported requirements changing up until several weeks before the launch of the website [NYTIMES].             |
+| SAM               | Partially | There were formal agreements made and used to manage all contractors but with a lack of quality control measures. Furthermore, the employees responsible for the awarding contracts were inexperienced. CMS did not comply with a federal rule requiring employees who oversee contracts worth more than $10 million to receive 96 hours of training [IHBEAT].            |
```

Comment:
There were formal agreements made and used to manage all contractors but with a lack of quality control measures. This could be a little bit more direct, for example: Formal agreements were in place to manage contractors, but these had a lack of quality control measures.